src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -537,7 +558,12 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
       _output << std::endl;
       count = 0;
       for (auto& tile : graph.get_child("tile")) {
-        _output << boost::format("Core [%2d]\n") % count++;
+        int curr_core = count++;
+        if(aieCoreList.size() && (std::find(aieCoreList.begin(), aieCoreList.end(),
+		     std::to_string(curr_core)) == aieCoreList.end()))
+          continue;
+
+        _output << boost::format("Core [%2d]\n") % curr_core;
         _output << fmt4("%d") % "Column" % tile.second.get<int>("column");
         _output << fmt4("%d") % "Row" % tile.second.get<int>("row");

```

Comment:
is there a reason "fmt4" was chosen? throughout xbutil reports, we are following indentation of 2